packages/legend-studio-app/scripts/setup.js

```diff
@@ -43,7 +43,7 @@ fs.writeFileSync(
       appName: 'studio',
       env: 'local',
       sdlc: {
-        url: 'https://legend.finos.org/sdlc',
+        url: 'https://legend.finos.org/sdlc/api',
       },
       engine: {
         url: 'https://legend.finos.org/exec',
```

Comment:
if that is the case for SDLC, why don't we have to do the same for EXEC?

Reply:
always been the case :) we should change our clients to handle the /api in building the url. Right now all calls to legend sdlc required the /api prefix.

Reply:
ok, let's not change the config setup here, let's change the client instead. Looks super weird from config perspective.